ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -52,24 +53,30 @@
     - zulu7                       # JDK8 Build Bootstrap
     - role: adoptopenjdk_install
       jdk_version: 8
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK11 Build Bootstrap
       jdk_version: 10
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install
       jdk_version: 11
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK14 Build Bootstrap
       jdk_version: 13
+      when: ansible_architecture != "riscv64"
       tags: build_tools
     - role: adoptopenjdk_install  # JDK15 Build Bootstrap
       jdk_version: 14
+      when: ansible_architecture != "riscv64"
```

Comment:
I'm tempted to suggest doing this within the role to avoid duplication ... Then we can adjust it as and when we get more versions available and it makes it explicit within the role

Reply:
I'm not sure about that, we will likely soon have a jdk11 release which means we'll need the ability to skip on anything but 11. Skipping within the role would mean that we skip all versions?

Reply:
Not really, as I said "we can adjust it as and when we get more versions available" within the role, that way the role could be re-used by others without having to be careful about how to call it, and it keeps the top level playbook cleaner.

Reply:
Swapfile re-enabled as per https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1483#issuecomment-733810048 - resolving